cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -134,8 +134,6 @@ if (EMSCRIPTEN_FORCE_COMPILERS)
 	set(CMAKE_CXX_COMPILER_ID Clang)
 	set(CMAKE_CXX_STANDARD_COMPUTED_DEFAULT 98)
 
-	set(CMAKE_C_COMPILER_TARGET "asmjs-unknown-emscripten")
```

Comment:
is this actually necessary? In particular, have you checked that CMake correctly infers the target when it queries the compiler? I wouldn't be at all surprised if its check doesn't work right due to the quirks of emscripten.